apps/studio/tests/unit/lib/editor.spec.js

```diff
@@ -0,0 +1,130 @@
+import {
+  makeDBHint,
+  parseDBHintTable,
+  parseDBHintTables,
+  queryTable,
+  findTablesBySchema,
+  pushTablesToResult,
+  sanitizeTableName,
+} from "../../../src/lib/editor";
+import { tableOrViews } from "../../fixtures/tables";
+
+describe("lib/editor", () => {
+  let dbHint;
+
+  beforeAll(() => {
+    dbHint = makeDBHint(tableOrViews, "postgresql");
+  });
+
+  describe("makeDBHint", () => {
+    it("should make arrays of tables and schemas", () => {
+      expect(makeDBHint(tableOrViews, "postgresql")).toEqual({
+        tables: [
+          { name: "my_table", schema: "public" },
```

Comment:
In postgres if you create a normal table using quotes, like `'"normal_table"'`, it always has to be referenced with the quotes I think? Does it work in that case?

(didn't test this, just pulling it from my memory)

Reply:
Yup. Table names that are case sensitive will need to be referenced with quotes. I'll add this to the test.